XenAdmin/TabPages/PhysicalStoragePage.cs

```diff
@@ -337,6 +336,41 @@ private void dataGridViewSr_SortCompare(object sender, DataGridViewSortCompareEv
                 e.SortResult = diff > 0 ? 1 : diff < 0 ? -1 : 0;
                 e.Handled = true;
             }
+            else if (e.Column.Index == columnName.Index)
+            {
+                sr1CompStr = sr1.Name();
+                sr2CompStr = sr2.Name();
+            }
+            else if (e.Column.Index == columnDescription.Index)
+            {
+                sr1CompStr = sr1.Description();
+                sr2CompStr = sr2.Description();
+            }
+            else if (e.Column.Index == columnType.Index)

```

Comment:
The shared and type columns do not need handling.

Reply:
Why ? Because there won't be numbers in the string?